packages/my_static_tf/src/static_at_tf_publisher_map1.py

```diff
@@ -19,7 +19,7 @@ def fetch_tag_id(tag) -> int:
 MAP_TRANSFORM_DICT = {
     31: ((0, 0, 0.075), (0, 0, 0)),
     32: ((0, 0.5, 0.075), (0, 0, 0)),
-    65: ((0.5, 0.25, 0), (0, 0, 0)),
+    65: ((0.5, 0.25, 0.075), (0, 0, 0)),
```

Comment:
@yosoufe, can you explain why you changed this value?

Reply:
I think this is fixing the mistake. Each tuple defines the transformation of AprilTags; the first component is the translation and the second component is rotation. I think I omitted the z-axis offset of AprilTag #65.

Reply:
This is  not related to fixing the TFs. We already discussed this in one of the office hours. Why one of the tags location has `z=0` and the others have `z=0.075` ? I think all of them should be at the height of `0.075`. Correct?

Reply:
Oh yeah you're right. When it's all in the same PR, I get confused.

Reply:
sorry, I had to clarify it.